### PR TITLE
Fix files NoneType issue

### PR DESCRIPTION
**Pull request by hanchuntao**
[OLCS-274](https://issues.redhat.com/browse/OLCS-274)

Fixes OLCS-274

### CODE REVIEW

**Comment by hanchuntao** [comment]
@rhyw @qduanmu @huiwangredhat PTAL

File: openlcsd/flow/tasks.py [hunk 1]
```diff
@@ -641,9 +641,7 @@ def deduplicate_source(context, engine):
                             os.remove(path)
                 else:
                     swhids = list(set(swhids))
-
-                if swhids:
-                    context['source_info']['swhids'] = swhids
+                context['source_info']['swhids'] = swhids if swhids else []
```
Comment (huiwangredhat) [inline openlcsd/flow/tasks.py:644]: Does that mean it allowed the swhids is None? When it will be None?

Comment (hanchuntao) [reply openlcsd/flow/tasks.py:644]: the existing code means if swhids exist, will set `context['source_info']['swhids'] = swhids`, but if not exist, will not be set, but in the backend, if not set, we will get `None` for swhids.

code:
https://github.com/RedHatProductSecurity/openlcs/blob/69b18fa2c290bc9358de7d3d7e58b2b69818d35f/openlcs/packages/views.py#L807

**Comment by huiwangredhat** [comment]
LGTM.

**Pull request merged**